src/lib/core/CHIPError.h

```diff
@@ -618,6 +618,7 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_INVALID_TLV_CHAR_STRING                     CHIP_CORE_ERROR(0x15)
 
+
```

Comment:
Again: why are there _ignored resolved review comments_ saying exactly what I was about to say?  This sort of thing is a waste of both my time and @mkardous-silabs's.

Reply:
The accepted suggestions were jeopardized by merge conflicts. I'll make sure to double check them going forward. Sorry about this.